packages/library/src/components/bal-select/option.utils.ts

```diff
@@ -0,0 +1,62 @@
+import { BalOptionController } from '../bal-select-option/bal-select-option'
+
+export function addOption(options: BalOptionController[], option: BalOptionController): BalOptionController[] {
+  return [...options, option]
+}
+
+export function updateOption(options: BalOptionController[], optionToUpdate: BalOptionController): BalOptionController[] {
+  const copyOfOptions = [...options]
+  for (let index = 0; index < copyOfOptions.length; index++) {
```

Comment:
Why not map?
=> returns new array
if (option.value === optionToUpdate.value) {
  return optionToUpdate
}
return option;

Reply:
used map